docs/competition/teams/role-descriptions/kit-return.md

```diff
@@ -19,35 +19,42 @@ rejected. When presented with a RUB full of kit, kit return staff should:
-    * Write down the part code of the Really Useful Box on the kit return form.
-    * Get their team leader to sign and write their name on the kit return form.
-
-* For teams keeping their kit (based on a list of teams authorised to do this):
-    * Write down the part code of the Really Useful Box on the kit return form.
-    * Give them two batteries, a battery charging bag and a charger+PSU. Record the part numbers of these five things on the kit return form.
-    * Get their team leader to sign and write their name on the kit return form.
-    * Give the team a return kit and ensure you inform them that they will be responsible for return shipping charges.
-    * Stick the appropriate 'authorised to keep kit' sheet onto the kit of the Really Useful Box with tape.
-
-* For each returned kit:
-    * Remove any batteries from the kit and store them separately
-    * Seal the handles of the kit with tape
-    * Stick a post-it on the end of the box with the team's TLA on it (to allow for double checking later on)
-    * Stack the box with the post-it visible
+For each returned kit:
 
+* Remove any batteries from the kit and store them separately
+* Seal the handles of the kit with tape
+* Stick a post-it on the end of the box with the team's TLA on it (to allow for double checking later on)
+* Stack the box with the post-it visible
+
+## Teams returning kit
+
+* Check the contents against a list of parts expected in the kit.
+  Make all reasonable attempts to retrieve kit there and then i.e. if they've left a bit upstairs they go and get it.
+* If they do not have something, record it on a missing kit form.
+* If the kit return form indicates that they have more than a normal kit to return (i.e. they had broken kit and did not return it upon arrival), see if they can return it now. If not, record it on a missing kit form.
+* Indicate if all parts are present or if a missing kit form has been filled in on the kit return form (provided by me).
+* Write down the part code of the Really Useful Box on the kit return form.
+* Get their team leader to sign and write their name on the kit return form.
+
+### Teams leaving before competition-end
+
+If a team wishes to leave before the end of the competition, we should attempt to find any missing kit, rather than recording it on a missing kit form.
+
+## Teams keeping kit
+
+* Write down the part code of the Really Useful Box on the kit return form.
+* Give them two batteries, a battery charging bag and a charger+PSU. Record the part numbers of these five things on the kit return form.
+* Get their team leader to sign and write their name on the kit return form.
+* Give the team a return kit and ensure you inform them that they will be responsible for return shipping charges.
+* Stick the appropriate 'authorised to keep kit' sheet onto the kit of the Really Useful Box with tape.
+* Kit should be shipped to the postal address listed on [the website](https://studentrobotics.org/contact/).
```

Comment:
Perhaps we could include this information as part of the sheet itself? (see also https://github.com/srobo/kit-coordination-docs/commit/2454c993f028301012fe1b2edf5de465a44e0f45#r32890277)

Reply:
This has been done.